include/iarc7_motion/PidController.hpp

```diff
@@ -42,19 +40,19 @@ class PidController
     void resetAccumulator();
 
 private:
-    const double p_gain_;
-    const double i_gain_;
-    const double d_gain_;
+    double& p_gain_;
```

Comment:
Why do these have to be references?

Reply:
By being references the actual memory locations are the same as the array of settings the is passed in and controlled by the dynamic reconfigure callback. It's poor practice but I don't think its worth fixing with a settings object. I'll do it you insist though.